docs/delegators/how-to-guides/bridge-lpt.md

```diff
@@ -32,7 +32,9 @@ Please note that you will need some ETH in your wallet to complete this guide. I
 ### Bridging from L2 to L1
 The same general instructions apply with three differences:
 - You should start with your wallet connected to Arbitrum (Arbitrum Rinkeby if using testnet)
-- You will need to manually approve the amount that you are bridging using the [Arbiscan UI](https://arbiscan.io/address/0x289ba1701C2F088cf0faf8B3705246331cB8A839#writeContract). To do so, connect your wallet, scroll to the Approve function, enter the L2LPTGateway address [0x6D2457a4ad276000A615295f7A80F79E48CcD318](https://arbiscan.io/address/0x6D2457a4ad276000A615295f7A80F79E48CcD318) and the amount you'll be transferring, and click "Write"
+- You will need to manually approve the amount that you are bridging using the [Arbiscan UI](https://arbiscan.io/address/0x289ba1701C2F088cf0faf8B3705246331cB8A839#writeContract). To do so, connect your wallet, scroll to the Approve function, enter the L2LPTGateway address [0x6D2457a4ad276000A615295f7A80F79E48CcD318](https://arbiscan.io/address/0x6D2457a4ad276000A615295f7A80F79E48CcD318) and the amount you'll be transferring and click "Write". Note that the amount is in "Wei", so use a [Unit Converter](https://etherscan.io/unitconverter) (e.g. if you want to bridge 10 LPT, you need to enter 10000000000000000000).
     - We are looking into options to remove this step. If it seems complex, we recommend swapping to ETH via [Uniswap](https://app.uniswap.org/#/swap?chain=arbitrum) and then transferring the ETH from L2 to L1 instead.
-- Enter the L2 LPT contract address ([0x289ba1701C2F088cf0faf8B3705246331cB8A839](/protocol/reference/deployed)) as the destination address on the [Arbitrum Bridge](https://bridge.arbitrum.io/)
+- Next, click on the arrow below the amount input. The button will switch from "Deposit" to "Withdraw"
+- Click on "Token" -> "View Token Lists" -> enable "Arbed CMC List" and click on "View Tokens" again. You should now be able to choose LPT in the list.
+- After you sent the withdraw tx, you need to wait ~1 week until you can claim your LPT on L1. You'll see the countdown below. Once the confirmation period is over, you can click "Claim" (you need to be connected to mainnet for that).
```

Comment:
typo: After you se**n**d

also a phrasing update:
```
you can connect to the Ethereum mainnet and click "Claim" to access your L1 LPT.
```

Reply:
sent seems correct to me - was meant to be in past tense. 
Agree on the phrasing change, maybe even:  `you can connect to the Ethereum mainnet and click "Claim" to withdraw your LPT on L1.`